CONTRIBUTING.md

```diff
@@ -59,33 +59,70 @@ pnpm i
 pnpm run setup
 ```
 
-Note for Windows:
-Use the latest version of [Git Bash](https://gitforwindows.org/)
+> 💡 For Windows users: use the latest version of [Git Bash](https://gitforwindows.org/).
 
 ## Building packages when you make changes
 
 In the root directory:
 
 - `pnpm run setup` will install and build all the packages
 - `pnpm -r run build` (-r for recursive) will build all the packages
-- `pnpm -r run dev` (-r for recursive) will build all the packages without running `tsc` (Fastest)
+- `pnpm -r run dev` (-r for recursive) will build all the packages without running `tsc`
+- `pnpm run watch` will build any package that has been modified without running `tsc` (Fastest)
 
 In a package directory, like `packages/client`:
 
 - `pnpm run build` will build the package
 - `pnpm run dev` will build the package without running `tsc` (Fastest)
 
-Note: Our builder is ESbuild
+> 💡 Our builder is ESbuild
 
 ## Prisma Client
 
 ### First contribution
 
-1. `cd packages/client`
-1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
-1. `cd fixtures/blog`
-1. `npx prisma db push --skip-generate` will create the database structure
-1. `ts-node main`
+Create a reproduction folder for developing, trying a new feature, or a fix.
+
+#### Setting up a locally-linked development folder
+
+We suggest that you create a `repros` folder outside of the `prisma` folder.  
```

Comment:
This says `outside of the `prisma` folder.` but I see a repro folder is contained in this PR, meaning it's inside prisma repo. I'm confused now

Reply:
Yes, basically I don't want people to work with their repros inside of the main repo and this one is basically a template (for now only `basic-sqlite`) that they can copy outside and work in.

Reply:
Any ideas about how to clarify that?

Reply:
Thinking about this:

It could also work to have a `reproductions` directory at the root like you added
- which we put in .gitignore but with one example and add a README
- we can also ignore changes in our GitHub Actions (so jobs are not triggered by a change in this directory) [like here](https://github.com/prisma/prisma/blob/main/.github/workflows/test.yml#L8) 
- we can point people (internally or externally or both) to create a repro by copying the example

Like this, there is no problem of having it inside the monorepo?